### PR TITLE
Add SMISMEMBER support

### PR DESCRIPTION
**Pull request by mathspanda**
Add support for [SMISMEMBER](https://redis.io/commands/smismember).

### CODE REVIEW

File: src/redis_cmd.cc [hunk 1]
```diff
@@ -1985,6 +1985,31 @@ class CommandSIsMember : public Commander {
   }
 };
 
+class CommandSMIsMember : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    Redis::Set set_db(svr->storage_, conn->GetNamespace());
+    std::vector<Slice> members;
+    for (unsigned int i = 2; i < args_.size(); i++) {
```
Comment (torwig) [inline src/redis_cmd.cc]: I'd recommend using the `size_t` type here instead of `unsigned int`.

Comment (git-hulk) [inline src/redis_cmd.cc]:
```suggestion
    for (size_t i = 2; i < args_.size(); i++) {
```

**Review by git-hulk — APPROVED** [review]

**Review by torwig — APPROVED** [review]
LGTM

**Comment by git-hulk** [comment]
Thanks to @mathspanda contribution and @torwig review, merging...

**Pull request merged**